python/lsst/summit/utils/astrometry/utils.py

```diff
@@ -0,0 +1,288 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import AltAz, EarthLocation, SkyCoord
+
+from lsst.afw.geom import SkyWcs
+from lsst.daf.base import PropertySet
+from lsst.pipe.tasks.characterizeImage import CharacterizeImageTask, CharacterizeImageConfig
+
+__all__ = [
+    'chuckHeaderToWcs',
+    'getAverageRaFromHeader',
+    'getAverageDecFromHeader',
+    'getAverageAzFromHeader',
+    'getAverageElFromHeader',
+    'genericCameraHeaderToWcs',
+    'getIcrsAtZenith',
+    'headerToWcs',
+    'runCharactierizeImage',
+    'filterSourceCatOnBrightest',
+]
+
+
+def chuckHeaderToWcs(exp, nominalRa=None, nominalDec=None):
+    """Given an exposure taken by Chuck at his house, construct a wcs
```

Comment:
I thought this was going to throw the header somewhere, but either way, someone, some day will surely be confused by the origin of this joke, at the least.

Reply:
looool 🤣 I had never seen it like that, and you're right, if you don't know Chuck that's actually really quite confusing. I will change that to Claver header I think!